### PR TITLE
chore(test): add macro performance tests for mergeAll

### PR DESCRIPTION
**Pull request by kwonoj**
- add test for mergeAll
- spec description correction to existing tests

Continue effort to cover #100.

### CODE REVIEW

**Comment by kwonoj** [comment]
Corrected typos.

File: perf/macro/merge/perf.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ var Rx2Merge = document.querySelector('#rx-2-merge');
 var RxNextMerge = document.querySelector('#rx-3-merge');
 
 var Rx2TestObservable = Rx.Observable.range(0, numIterations);
-var RxNextTestObservable = RxNext.Observablerange(0, numIterations);
+var RxNextTestObservable = RxNext.Observable.range(0, numIterations);
```
Comment (kwonoj) [inline perf/macro/merge/perf.js]: all typo error comes from this, copy-paste without giving enough caution. Sorry again for error, will keep more eye in further.

Comment (blesh) [inline perf/macro/merge/perf.js]: No problem, I do stuff like that all the time.

**Comment by blesh** [comment]
merged with 401fe1f

Thanks for your hard work @kwonoj!

**Pull request closed without merging**